src/components/Header.js

```diff
@@ -1,10 +1,61 @@
-import React from 'react';
-import styled from 'styled-components';
+import React from "react";
+import styled from "styled-components";
+import { images } from "./images/image";
 
-export default function Header() {
-	return <Wrapper>발신자 변경해주는 상단바</Wrapper>;
+export default function Header(props) {
+  //change parent's user state using props
+  const handleClick = () => {
+    props.onClick();
+  };
+
+  return (
+    <Profile onClick={handleClick}>
+      {/* if user value is false */}
+      {/* I hate this code!!!!!! I want simpler code... */}
+      {!props.user && (
+        <>
+          <img src={images.COOL} />
+          <div>
+            <h2 style={{ marginBottom: "0" }}>정쿨</h2>
+            <br />
+            현재 활동중
+          </div>
+        </>
+      )}
+      {/* if user value is true */}
+      {props.user && (
+        <>
+          <img src={images.EUNKO} />
+          <div>
+            <h2 style={{ marginBottom: "0" }}>고은</h2>
+            <br />
+            현재 활동중
+          </div>
+        </>
+      )}
+    </Profile>
+  );
 }
-const Wrapper = styled.div`
-	background-color: black;
-	color: white;
+
+const Profile = styled.div`
+  background-color: white;
+  display: flex;
+  align-items: center;
+  width: 100%;
+
+  div {
+    margin-left: 10px;
+  }
+
+  img {
+    border-radius: 20px;
+    width: 100px;
+    height: 100px;
+  }
+
+  padding-left: 20px;
+  height: 120px;
+  z-index: 1;
+  position: fixed;
+  opacity: 0.8;
```

Comment:
이렇게 하면 전체에 opacity를 줘버려서 프로필사진까지 투명도가 생기네요ㅠㅠ
위에 `background-color: white;`를 지우시고 background에만 opacity를 이렇게 줄 수 있답니당

```suggestion
  background-color: rgba(255, 255, 255, 0.8);
```

Reply:
생각지도 못한 부분이네요!! 감사합니다 :)